context/app/static/js/components/Search/config.js

```diff
@@ -27,7 +34,7 @@ export const sampleConfig = {
     filter('mapped_specimen_type', 'Specimen Type'),
     filter('donor.group_name', 'Group'),
     filter('created_by_user_displayname', 'Creator'),
-  ],
+  ].concat(makeDonorMetadataFilters(false)),
```

Comment:
@ngehlenborg : This and the one below add the four donor filters.

Reply:
Is this for sample only or also for datasets? We want it for both, I don't remember if I said that on Slack.

Reply:
Both: should have been more clear.

Reply:
Thanks!